feature/preview/src/main/java/com/google/jetpackcamera/feature/preview/ui/CameraControlsOverlay.kt

```diff
@@ -249,7 +249,7 @@ private fun CaptureButton(
             multipleEventsCutter.processEvent {
                 when (previewMode) {
                     is PreviewMode.StandardMode -> {
-                        onCaptureImage()
+                        onCaptureImageWithUri(context.contentResolver, null) {}
```

Comment:
In standard mode, when this completes successfully, we should broadcast `ACTION_NEW_PICTURE` with the captured URI. That means you should be returning the Uri from this method as well.

Reply:
Updated. Instead of having the method returning uri, I made it so that ImageSaved event has the savedUri, which can be picked up by the callback in PreviewMode.StandardMode in the MainActivity for broadcasting